src/av2/evaluation/tracking/utils.py

```diff
@@ -93,6 +94,20 @@ def load(path: str) -> Any:  # noqa
         return pickle.load(f)
 
 
+def annotate_frame_metadata(prediction_frames: Frames, label_frames: Frames, metadata_keys: List[str]) -> None:
```

Comment:
[Nit] Use the most generic type possible for args.
```suggestion
def annotate_frame_metadata(prediction_frames: Frames, label_frames: Frames, metadata_keys: Sequence[str]) -> None:
```